### DISCUSSION TITLE
querying the extra columns in many to many relation table

### DISCUSSION THREAD

**Discussion by muratcankaracabey** [supabase/supabase#26755] [Questions]
I have a basic many to many table

`item_collection`

which has columns

`item_id, collection_id, extra_column`

however when I query

`await client.from('collection').select('*, item(*)')`

I can not get the extra_column. Is there a way to do this or should I write a custom function for this which is too cumbersome.

**Comment by GaryAustin1** [comment]
`.select('*,item(*),item_collection(extra_column)')`
